manifests/server/apache.pp

```diff
@@ -47,17 +25,34 @@
     purge_configs  => $purge_configs,
   }
 
+  # TODO Audit this list. Drupal actually doesn't need most of these.
```

Comment:
This list actually came from acquia - this was what they were installing at the time. The list was acquired for probo's environment building which was what lead to the creation of this module in the first place.